packages/components/input/package.json

```diff
@@ -36,7 +36,7 @@
   "peerDependencies": {
     "react": ">=18",
     "react-dom": ">=18",
-    "@nextui-org/theme": ">=2.1.0",
+    "@nextui-org/theme": ">=2.1.1",
```

Comment:
current theme version is 2.2.11. if you apply minor on theme, then it should be `>=2.3.0` instead.